Pythonize_Everything/auto_back_up.py

```diff
@@ -40,7 +40,7 @@ def size_if_newer(source, target):
 
     # The time difference of one second is necessary since subsecond accuracy
     # of os.st_mtime is striped by copy2
-    return src_stat.st_size if (src_stat.st_mtime - target_ts > 1) else False
+    return src_stat.st_size if abs(src_stat.st_mtime - target_ts > 1) else False
```

Comment:
@kumar-mahendra I think you meant to say the below line :-
```return src_stat.st_size if (abs(src_stat.st_mtime - target_ts) > 1) else False```

Taking the absolute first and then comparing!